index.js

```diff
@@ -29,7 +32,7 @@ const encodingMethods = {
  */
 
 module.exports = (options = {}) => {
-  let { filter = compressible, threshold = 1024 } = options
+  let { brotliOptions, filter = compressible, threshold = 1024 } = options
```

Comment:
i'd prefer if it were just `options.brotli` but i will leave it

Reply:
I am totally open to change it. Please think about it and if that's your preference I will rename.

Personally I don't like `brotliOptions` either &mdash; too long.